.swiftlint.yml

```diff
@@ -1,6 +1,6 @@
-line_length:
-  warning: 150
-  ignores_comments: true
 
 disabled_rules:
-  identifier_name
+- identifier_name
+- trailing_whitespace
+- line_length
+- variable_name
```

Comment:
I still think your SwiftLint might be outdated. 
`variable_name` has been deprecated. This SwiftLint doesn't compile for me 🤔
Other than this, LGTM